### DISCUSSION TITLE
Can't reach database server at `aws.connect.psdb.cloud`:`3306`

### DISCUSSION THREAD

**Discussion by khanhthanhdev** [prisma/prisma#19531] [Q&A]
I use: `npm run dev `and get error:
Unhandled Runtime Error
Error: 
Invalid `prisma.post.findMany()` invocation:

Can't reach database server at `aws.connect.psdb.cloud`:`3306`

Please make sure your database server is running at `aws.connect.psdb.cloud`:`3306`.

`DATABASE_URL='mysql://5monfhha5p31sea64hkc:************@aws.connect.psdb.cloud/blog-ai-auto?ssl={"rejectUnauthorized":true}&&sslaccept=strict'
`
how can i solve this problem

**Comment by ludralph** [comment]
Hi @khanhthanhdev 👋 

1. Can you please make sure your AWS instance has open ports and firewall is not blocking inbound connection. Try connecting to the database using cli or a gui first.
2. If the above step work correctly, you can try adding  `?connect_timeout=30&pool_timeout=30&socket_timeout=30` (or higher seconds) to DATABASE_URL

**Reply by ludralph** [reply]
Hi @khanhthanhdev 

do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏

**Comment by mcrstudio** [comment]
Just a note to anybody seeing this - if you're using a public network, there's a good chance you'll see this error. I was on a train and got this error using their public WiFi but when switching over to my phones hotspot, error gone.

**Reply by nurul3101** [reply]
Thanks for sharing! 🙂
